std_msgs/Header header
uint32 hertz
string message
